File: Week 1_ C/Problems/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Create height variable
    int height = 0;

    do
    {
        // Getting an input
        height = get_int("Height: ");
    }
    // Validating the input
    while (height < 1 || height > 8);

    // For each row
    for (int row = 0; row < height; row++)
    {
        // For the spaces before the stairs
        for (int i = 0; i < height - row - 1; i++)
        {
            printf(" ");
        }

        // For the first set of stairs
        for (int i = 0; i <= row; i++)
        {
            printf("#");
        }

        // For the space in between the stairs
        printf("  ");

        // For the second set of stairs
        for (int i = 0; i <= row; i++)
        {
            printf("#");
        }

        // Move to the next row
        printf("\n");
    }
}

File: Week 6_ Python/Problems/sentimental-mario-more/mario.py
# Get height input
while True:
    try:
        height = int(input("Height: "))
        if height >= 1 and height <= 8:
            break
    except ValueError:
        print(end='')

# For each row
for i in range(height):
    # Spaces
    print(" " * (height - i - 1), end="")

    # Left stair
    print('#' * (i + 1), end="  ")

    # Right stair
    print("#" * (i + 1))